### PR TITLE
Task for new AMANDA query

### PR DESCRIPTION
**Pull request by Charlie-Henry**
## Associated issues
Adding new task for helping to track review time for ROW 
https://github.com/cityofaustin/atd-data-tech/issues/14143

## Associated repo
https://github.com/cityofaustin/atd-executive-dashboard/blob/main/AMANDA/amanda_to_s3.py

## Testing
**Steps to test:**
Trigger this DAG and check it completes ✅, if you're running this on a mac you will probably get the same error I'm getting `Cannot locate a 64-bit Oracle Client library` 😑 so I'm unsure how to test this without yolo'ing.

---
#### Ship list
- [ ] Code reviewed 
- [ ] Product manager approved
- [ ] Add note to 1PW secrets moved to API vault and check for duplicates

### CODE REVIEW

File: dags/atd_executive_dashboard_row_weekly_summary.py [hunk 1]
```diff
@@ -17,7 +19,7 @@
     "email_on_failure": False,
     "email_on_retry": False,
     "retries": 0,
-    "execution_timeout": duration(minutes=5),
+    "execution_timeout": duration(minutes=30),
```
Comment (Charlie-Henry) [inline dags/atd_executive_dashboard_row_weekly_summary.py:22]: unlikely to take this long but want to be safe

**Review by mddilley — APPROVED** [review]
yolo! 🚀 The new task looks great. 🚢

File: dags/atd_executive_dashboard_row_weekly_summary.py [hunk 2]
```diff
@@ -179,4 +181,17 @@
...
+        command=f"python AMANDA/amanda_to_s3.py --query ex_permits_issued",
+        environment=env_vars,
+        tty=True,
+        force_pull=False,
+        mount_tmp_dir=False,
+        retries=3,
+        retry_delay=duration(seconds=60),
+    )
+
+    t1 >> t2 >> t3 >> t4 >> t5 >> t6 >> t7
```
Comment (johnclary) [inline dags/atd_executive_dashboard_row_weekly_summary.py:197]: @Charlie-Henry i can't remember if we've talked about this, or if it is even desired in this case, but it is possible to use [trigger rules](https://airflow.apache.org/docs/apache-airflow/1.10.10/concepts.html#trigger-rules) to enable downstream tasks to run regardless of if an earlier task fails.

**Review by johnclary — COMMENTED** [review]
👍

File: dags/atd_executive_dashboard_row_weekly_summary.py (hunk 2 above)
Comment (Charlie-Henry) [reply dags/atd_executive_dashboard_row_weekly_summary.py:197]: ah yes you did comment this before, good idea. I'll see if I can get something to work

**Pull request merged**